firstpartydata/first_party_data_test.go

```diff
@@ -476,7 +476,7 @@ func TestExtractBidderConfigFPD(t *testing.T) {
 		t.Run(test.Name(), func(t *testing.T) {
 			filePath := testPath + "/" + test.Name()
 
-			fpdFile, err := loadFpdFile(filePath)
+			fpdFile, err := loadFpdFile[fpdFile](filePath)
```

Comment:
Nitpick: `fpdFile` that is returned should probably have different name to avoid confusion with the `fpdFile` struct. 
Something like: `testFile, err := loadFpdFile[fpdFile](filePath)`